traitcuration/traits/datasources/ols.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+
+BASE_URL = "https://www.ebi.ac.uk/ols/api/"
+
+
+def make_ols_query(term_uri, ontology):
+    """
+    Takes in a term URI (or IRI as referenced in OLS documentation) and the ontology id to search against. Returns a
+    dictionary for that term with the fields 'label', 'obo_id' which is the CURIE and 'status'.
+    """
+    results = requests.get(
+        f"{BASE_URL}ontologies/{ontology}/terms?iri={term_uri}").json()
```

Comment:
```suggestion
    results = requests.get("{BASE_URL}ontologies/{ontology}/terms?iri={term_uri}").json()
```

Looks like this can fit easily into <120 character line. Is it at all possible that a linter you use for Python misbehaves and has its line limit set to the default 80, producing these things from time to time?

Reply:
Yes, this is caused by autopep8. I thought I had fixed that, but now it seems it's fixed for good, sorry for the inconvenience